dev-packages/e2e-tests/test-applications/aws-lambda-layer-cjs/tests/basic.test.ts

```diff
@@ -18,7 +18,7 @@ test('Lambda layer SDK bundle sends events', async ({ request }) => {
   );
 
   child_process.execSync('pnpm start', {
-    stdio: 'ignore',
+    stdio: 'pipe',
```

Comment:
```suggestion
    stdio: 'inherit',
```

not 100% sure but I think this is a more straightforward way?